odl/solvers/functional/functional.py

```diff
@@ -822,6 +823,83 @@ def __str__(self):
                                           self.translation)
 
 
+class InfimalConvolution(Functional):
+
+    """The functional representing ``h(x) = inf_y f(x-y) + g(y)``."""
```

Comment:
> Done. What is 
> Edit @kohr-h: fixed formatting
> supposed to mean?

It means he edited my comment and by courtesy he also mentioned that he did it (so I won't wake up and think I'm crazy and that my comments magically change).